src/MaaCore/Task/Reclamation/ReclamationCraftTaskPlugin.h

```diff
@@ -26,8 +26,8 @@ class ReclamationCraftTaskPlugin : public AbstractReclamationTaskPlugin
         Hold = 1
     };
 
-    std::vector<std::string> tools_to_craft;               // 要组装的支援道具
```

Comment:
@Alan-Charred I didn't actually notice this lol.
If we initialize the vector like this. Nothing works :P

Reply:
I'll temp fix by not initializing. Do we actually need initialization?